apps/stopwatch/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { InstantApp } from "@felvin-search/core";
+import { Component, queryToData } from "./App";
+
+const App: InstantApp = {
+  id: "@felvin-search-apps/stopwatch",
+  name: "stopwatch",
+  description: "it is a simple stopwatch",
+  queryToData,
+  Component,
+  // screenshotPath: "./files/screenshot.png",
+  // exampleSearchQueries: [],
```

Comment:
Thanks for adding the screenshot, we can add the example search query i.e. "stopwatch" and uncomment these lines.

Reply:
Okay mentor, I have uncommented those lines in my new commits.